demisto_sdk/commands/common/tools.py

```diff
@@ -4386,7 +4386,9 @@ def get_pack_latest_rn_version(pack_path: str) -> str:
         (str): The lastest version of RN.
     """
     list_of_files = glob.glob(pack_path + "/ReleaseNotes/*")
-    list_of_release_notes = [Path(file).name for file in list_of_files]
+    list_of_release_notes = [
+        Path(file).name for file in list_of_files if Path(file).suffix == ".md"
```

Comment:
This will actually make the issue more precise, we want to ensure there's a .md file with the specific version, we don't want it to work if we have only .json file.